core/src/react-server.ts

```diff
@@ -0,0 +1 @@
+export { unstable_serialize } from './use-swr'
```

Comment:
Still need to create a standalone file (`serialize.ts`) and export it from there, because `use-swr.ts` can have `useState` etc imported that are not allowed in Server Components.